### PR TITLE
Fix log permission to upload by linux XDP

### PR DESCRIPTION
**Pull request by ami-GS**
## Description

Linux XDP failure log is owned by root which prevent uploading it.

## Testing

See any failure happens on CI

## Documentation

N/A

### CODE REVIEW

File: src/platform/unittest/DataPathTest.cpp [hunk 1]
```diff
@@ -427,7 +427,7 @@ QuicAddr DataPathTest::UnspecIPv4;
 QuicAddr DataPathTest::UnspecIPv6;
 
 struct CxPlatDataPath {
-    QUIC_EXECUTION_CONFIG DefaultExecutionConfig { QUIC_EXECUTION_CONFIG_FLAG_XDP, 0, 0, {0} };
+    QUIC_EXECUTION_CONFIG DefaultExecutionConfig { QUIC_EXECUTION_CONFIG_FLAG_NONE, 0, 0, {0} };
```
Comment (nibanks) [inline src/platform/unittest/DataPathTest.cpp:430]: On purpose?

Comment (ami-GS) [reply src/platform/unittest/DataPathTest.cpp:430]: Yes, to generate any error

File: .github/workflows/stress.yml [hunk 2]
```diff
@@ -127,6 +127,10 @@ jobs:
       timeout-minutes: 65
       shell: pwsh
       run: scripts/spin.ps1 -AZP -Config ${{ matrix.vec.config }} -Arch ${{ matrix.vec.arch }} -Tls ${{ matrix.vec.tls }} -Timeout ${{ env.main-timeout }} -RepeatCount ${{ env.main-repeat }} -AllocFail ${{ env.main-allocfail }} ${{ matrix.vec.xdp }}
+    - name: Fix log permissions for Linux XDP
+      if: (matrix.vec.plat == 'linux' && matrix.vec.xdp == '-UseXdp')
+      run: |
+        sudo chmod -R 777 artifacts
```
Comment (ami-GS) [inline .github/workflows/stress.yml:133]: Why this `if` doesn't work!?!?!?!?

Comment (ami-GS) [reply .github/workflows/stress.yml:133]: @nibanks @csujedihy 
Any idea?????

**Comment by ami-GS** [comment]
ready to go if anyone doesn't know the `if: (matrix.vec.plat == 'linux' && matrix.vec.xdp == '-UseXdp')` solution as mentioned above

**Review by nibanks — APPROVED** [review]

**Pull request merged**